AsyncDisplayKit/Layout/ASStackLayoutDefines.h

```diff
@@ -85,6 +85,22 @@ typedef NS_ENUM(NSUInteger, ASStackLayoutAlignSelf) {
   ASStackLayoutAlignSelfStretch,
 };
 
+// TODO documentation
+typedef NS_ENUM(NSUInteger, ASStackLayoutFlexWrap) {
+  ASStackLayoutFlexWrapNoWrap,
+  ASStackLayoutFlexWrapWrap,
```

Comment:
How about `ASStackLayoutFlexWrapDisable` and ``ASStackLayoutFlexWrapEnable` to avoid the double `Wrap`?